client/helpers_test.go

```diff
@@ -864,8 +864,8 @@ func TestApplyChangelistTargetsToMultipleRoles(t *testing.T) {
 	assert.False(t, ok, "no change to targets/level2, so metadata not created")
 }
 
-// ApplyTargets falls back to role that exists when adding or deleting a change
-func TestApplyChangelistTargetsFallbackRoles(t *testing.T) {
+// ApplyTargets does not fall back to role that exists when adding or deleting a change to a nonexistent delegation
+func TestApplyChangelistTargetsDoesNotFallbackRoles(t *testing.T) {
```

Comment:
Agreed, I think I just changed the comments for myself/the team but it would make more sense when reading the code for the first time not to even include it :+1: